runelite-client/src/main/java/net/runelite/client/plugins/osrswikia/OSRSWikiaConfig.java

```diff
@@ -0,0 +1,22 @@
+package net.runelite.client.plugins.osrswikia;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+        keyName = "osrsWikia",
+        name = "Old School Runescape Wikia",
+        description = "Configuration for the Old School Runescape Wikia plugin"
+)
+public interface OSRSWikiaConfig extends Config
```

Comment:
Why is this config file needed? As it contains only 1 toggle that toggles entire functionality. Just remove config.

Reply:
Knew this would come up. I questioned whether or not it would make sense to move these trivial commands somewhere else for fear of bloating the plugin pane.

Reply:
Generally for this we have no config at all and instead flip the plugin on and off.

Reply:
Understood. Removed the config.